app/screens/AddEmission/AddEmissionScreen.tsx

```diff
@@ -6,7 +6,19 @@ import { TransportEnum, FoodEnum, StreamingEnum } from "carbon-footprint";
 import DateTimePickerModal from "react-native-modal-datetime-picker";
 import { KeyboardAwareScrollView } from "react-native-keyboard-aware-scroll-view";
 
-import { Text, Tag, TextInput } from "../../components";
+import { Text, Tag, TextInput } from "components";
+import { userPreferences } from "ducks";
+import { EmissionType } from "interfaces";
+import { EmissionPayload } from "interfaces/emission/emission.interface";
```

Comment:
`import { EmissionType, EmissionPayload } from "interfaces";`
To be able to do that you simply need to import 2 times in index.ts files inside `interface`